ts/receiver/closedGroups.ts

```diff
@@ -1,214 +1,10 @@
-import _, { isEmpty, isNumber, toNumber } from 'lodash';
+import { isNumber } from 'lodash';
```

Comment:
If `sentAtMoreRecentThanWrapper` is the only function here then I think we should either rename the file or move the function into a related utils file?

Reply:
renamed